### DISCUSSION TITLE
This is a test discussion

### DISCUSSION THREAD

**Discussion by bsaris-ut** [user-interviews/ui-design-system#109] [General]
Just testing this out -- seeing how it works.

Btw, have I ever mentioned how much I love working with @jhforster?

**Comment by bsaris-ut** [comment]
One problem I see -- this doesn't seem to allow for any sort of commenting on the body or threading....

**Reply by bsaris-ut** [reply]
Oh -- you can thread on comments though!

**Reply by bsaris-ut** [reply]
But -- as is tradition -- can't thread on threads.

**Reply by rroppolo** [reply]
Interesting! I wish you could add labels to discussions

**Reply by bsaris-ut** [reply]
There are categories -- but I agree those seem heavier. I imagine they will add labeling in the future since that seems like a common feature.

**Reply by AlexAlt** [reply]
This isn't really formatted or organized in a way that is easy to parse- I would maybe hope this could replace Docs for initial requirements gathering/implementation discussions but not yet.

**Reply by jhforster** [reply]
WHAT AM I DOING AND HOW DOES THIS WORK?

**Reply by bsaris-ut** [reply]
> This isn't really formatted or organized in a way that is easy to parse- I would maybe hope this could replace Docs for initial requirements gathering/implementation discussions but not yet.

I agree -- ironically the discussion part is probably the least usable part of this. It may be possible that we need to learn to use the quote reply concept more...

**Comment by bsaris-ut** [comment]
cc @jhforster @hotcheetoqueen

**Reply by hotcheetoqueen** [reply]
hi

**Comment by rroppolo** [comment]
Also a reminder this is a public repo so all the world can see this :p
